src/WebUI/ImageDetails/ImageDetails.tsx

```diff
@@ -61,6 +62,9 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
                             {this._displayImageName}
                         </HeaderTitle>
                     </HeaderTitleRow>
+                    <HeaderDescription className={"text-ellipsis"}>
+                        {jobName}
+                    </HeaderDescription>
```

Comment:
we should addp padding at the end of the image details, not in the layers, but in the "Image details" section, please pod overview page.

Reply:
Top and bottom padding of 5px is added via class image-o-v-col-content..Its the same in Pod Overview